subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/plugins/JacocoPlugin.java

```diff
@@ -179,7 +179,7 @@ public void execute(JacocoReport reportTask) {
     }
 
     private void configureJacocoReportDefaults(final JacocoPluginExtension extension, final JacocoReport reportTask) {
-        reportTask.getReports().all(new Action<ConfigurableReport>() {
+        reportTask.getReports().withType(ConfigurableReport.class, new Action<ConfigurableReport>() {
```

Comment:
I made the change based your comment here: https://github.com/gradle/gradle/pull/1452#discussion_r106839189. Or I totally misunderstood your comment. Changed back.